README.md

```diff
@@ -98,6 +98,20 @@ origin  https://github.com/YOUR_GITHUB_USER_NAME/stem-diverse-tv (push)
 upstream        https://github.com/anitab-org/stem-diverse-tv (fetch)
 upstream        https://github.com/anitab-org/stem-diverse-tv (push)
 ```
+## YouTube Data API Enabling
+
+**Steps**
+
+- You need a [Google Account](https://www.google.com/accounts/NewAccount) to access the Google API Console, request an API key, and register your application.
```

Comment:
@s4-2 As these steps follow a certain order, this list should be a numbered list instead of bullet points.

Reply:
Okay